From c66d83876e33bc1902ea723c07479f8ed7d0f6aa Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 25 Jun 2024 17:38:16 +0200
Subject: [PATCH 0940/1695] drivers: stm32_rif: remove unused macros

Remove SCID_SHIFT, SEMWL_SHIFT and RIF_ID_SHIFT that shouldn't
supersede the ones above.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git fa4034554488cf7ac29a328024444ff1e8278f9d drivers: stm32_rif: remove unused macros]
Change-Id: Ia42c922a645bebb4192c1229ae6e39c2d829bbfe
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/389583
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/include/drivers/stm32_rif.h | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/core/include/drivers/stm32_rif.h b/core/include/drivers/stm32_rif.h
index 5130d70a2..526f0040c 100644
--- a/core/include/drivers/stm32_rif.h
+++ b/core/include/drivers/stm32_rif.h
@@ -34,17 +34,6 @@
 #define MAX_CID_BITFIELD		U(3)
 #define MAX_CID_SUPPORTED		U(7)
 
-#define SCID_SHIFT			U(4)
-#define SEMWL_SHIFT			U(16)
-#define RIF_ID_SHIFT			U(24)
-
-#define RIF_ID_MASK			GENMASK_32(31, 24)
-#define RIF_CHANNEL_ID(x)		((RIF_ID_MASK & (x)) >> RIF_ID_SHIFT)
-
-#define RIFPROT_SEC			BIT(8)
-#define RIFPROT_PRIV			BIT(9)
-#define RIFPROT_LOCK			BIT(10)
-
 #define _RIF_FLD_PREP(field, value)	(((uint32_t)(value) << \
 					  (field ## _SHIFT)) & \
 					 (field ## _MASK))
@@ -53,7 +42,7 @@
 					 (field ## _SHIFT))
 
 #define SCID_OK(cidcfgr, scid_m, cid)	(((cidcfgr) & (scid_m)) ==	\
-					 ((cid) << (SCID_SHIFT)) &&	\
+					 ((cid) << (_CIDCFGR_SCID_SHIFT)) && \
 					 !((cidcfgr) & (_CIDCFGR_SEMEN)))
 
 #define SEM_EN_AND_OK(cidcfgr, wl)	(((cidcfgr) & _CIDCFGR_CFEN) &&  \
-- 
2.47.2

